src/utils/json-table.ts

```diff
@@ -0,0 +1,53 @@
+import fs from 'fs';
+
+export class JsonTable<E> {
```

Comment:
Simple wrapper for our JSON commands.

An interesting thing you may take from it: the generic of `<E>`, allows us to use typescript for the input/return values in our helpers here!